.pre-commit-config.yaml

```diff
@@ -102,7 +102,7 @@ repos:
       stages: [pre-commit]
     - id: kw-commit-msg
       name: 'kw commit-msg'
-      entry: 'Utilities/Hooks/kw-commit-msg.py'
+      entry: 'Utilities/Hooks/run-pixi-python.sh Utilities/Hooks/kw-commit-msg.py'
       language: system
```

Comment:
I think the only change need it to change language: system -> language: python. The pre-commit environment already know the python environment. This is consistent with the reference hooks: https://github.com/pre-commit/pre-commit-hooks/blob/main/.pre-commit-hooks.yaml

Reply:
Just wondering, would it be sufficient to just explicitly mention `python`? As in:

      entry: 'python Utilities/Hooks/kw-commit-msg.py'

At least, the computers I tried (Windows) know how to run python from a command-line!

Reply:
>  system -> language: python

> just explicitly mention python

These are not going to work as robustly across all systems. The reference pre-commit python hooks set up their own Python environment. In general, we can not assume that Python is installed and available in the PATH.